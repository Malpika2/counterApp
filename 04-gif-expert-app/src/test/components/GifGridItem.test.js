import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import { GifGridItem } from '../../components/GifGridItem';

describe('Debe imostrar el componente correctamente', () => {

    let wrapper = shallow(<GifGridItem/>);

    test('Probar componenete GifGridItem', () => {
        
        expect(wrapper).toMatchSnapshot();    
    })
    
    
});
