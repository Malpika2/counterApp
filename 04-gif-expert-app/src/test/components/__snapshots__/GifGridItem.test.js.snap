// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debe imostrar el componente correctamente Probar componenete GifGridItem 1`] = `
<div
  className="card animate__animated animate__fadeIn "
>
  <img />
  <br />
</div>
`;
